modules/ROOT/pages/_partials/gs-accessing-prometheus-grafana-proc.adoc

```diff
@@ -0,0 +1,20 @@
+[id='gs-accessing-prometheus-grafana-proc']
+
+ifdef::env-github[]
+:imagesdir: ../images/
+endif::[]
+
+= Accessing Prometheus and Grafana
+
+Prometheus is a storage backend for platform metrics and Grafana acts as the interface for analysis and visualization. Prometheus and Grafana are available to monitor your platform. 
```

Comment:
You can use Prometheus and Grafana to monitor your platform?

Reply:
Removing this line.